pkgs/misc/emulators/emu2/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "emu2";
+  version = "unstable-2020-06-04";
```

Comment:
Maybe it's worth asking upstream if they're willing to tag releases?

Reply:
There are no release so there is nothing to tag :-)

Reply:
Maybe they'll start doing releases if enough people ask ;).

If you could squash the commits into one, I'll re-review the PR.